using-mobile-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -22,64 +25,73 @@ public static void syncIdentifier(final String identifierType,
 ```
 
 ### syncIdentifiers \(overloaded\)
 
-**Tip**: All given customer IDs are given the default authentication state of `UNKNOWN`.
+This  API  `syncIdentifiers()` is an overloaded version that does not include the parameter for the Authentication State, and assumes a default value of `VisitorID.AuthenticationState.UNKNOWN`
 
-These IDs are preserved between app upgrades, are saved and restored during the standard application backup process, and are removed at uninstall. If the current SDK privacy status is `optedout`, calling this method results in no operations being performed.
-
-{% hint style="info" %}
-The `identifiers` dictionary contains IDs with the Identifier type as the key, and the string identifier as the value. If the identifier type and identifier do not have valid string values \(non null or empty\), the identifier is ignored by the Identity extension.
-{% endhint %}
+These IDs are preserved between app upgrades, are saved and restored during the standard application backup process, and are removed at uninstall. 
 
 #### **Syntax**
 
 ```java
 public static void syncIdentifiers(final Map<String, String> identifiers);
 ```
 
-#### **Example**
+o  *identifiers* is a map that contains the Identifiers with the Identifier type as the key, and the string identifier as the value. If the identifier type and identifier do not have valid string values \(non null or empty\), the identifier is ignored by the Identity extension.
+
+#### Example
 
 ```java
 Map<String, String> identifiers = new HashMap<String, String>();
-identifiers.put("idType", "idValue");
+identifiers.put("idType1", "idValue1");
+identifiers.put("idType2", "idValue2");
+identifiers.put("idType3", "idValue3");
 Identity.syncIdentifier(identifiers);
 ```
 {% endtab %}
 
 {% tab title="iOS" %}
-### syncIdentifier
 
-Updates the provided customer ID with the Adobe Experience Cloud ID Service.
+### syncIdentifier<a id="syncIdentifier-ios"></a>
 
-This API synchronizes the provided customer identifier type key and value with the provided [authentication state](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/identity/identity-api-reference#authenticationstate) to the Adobe Experience Cloud ID Service. If this customer ID type exists in the service, this type is updated with the new ID and authentication state. Otherwise a new customer ID is added. This ID is preserved between app upgrades, is saved and restored during the standard application backup process, and is removed at uninstall. If the current SDK privacy status is `optedout`, calling this method results in no operations being performed.
+This `syncIdentifier()` API updates or appends the provided customer identifier type key and value with the given authentication state to the Adobe Experience Cloud ID Service. If the given customer ID type already exists in the service, then it is updated with the new ID and authentication state. Otherwise a new customer ID is added.
```

Comment:
- Please update the first sentence to read "This API updates..." and remove "`syncIdentifier()`".